### DISCUSSION TITLE
Can I make an egui::widgets::Image from Vec<u8>?

### DISCUSSION THREAD

**Discussion by Crispy13** [emilk/egui#5417] [Q&A] [answered]
I searched the docs and found `from_bytes`.

Is this the one?

but what value should I give to `uri` arg?

\* I made `Vec<u8>` with `ImageReader` of **image** crate

**Comment by lucasmerlin** [comment, marked as answer]
> but what value should I give to uri arg?

I believe you can use any you like, it just needs to be unique for your image

**Reply by Crispy13** [reply]
Thanks
